plugin/cmp.lua

```diff
@@ -122,7 +122,7 @@ if vim.on_key then
   end, vim.api.nvim_create_namespace('cmp.plugin'))
 end
 
-vim.cmd [[command! CmpStatus lua require('cmp').status()]]
+vim.api.nvim_create_user_command('CmpStatus', require('cmp').status, {})
```

Comment:
Thanks for helping migrate to neovim 0.7.x APIs, but one downside of lua-based command definition is it will make debugging a bit hard. You can try `:Telescope commands` to see its definition reads like:
```
CmpStatus         0                        <Lua function 78>          
```
Meh, what is `<Lua function 78>`? So I would recommend adding `desc = ...` in the opts table.

Reply:
Thanks for this! I didn't even think about that. I just updated my commit to include a description. Let me know if you want me to change the wording of it!